src/tsolvers/lasolver/CutCreator.cc

```diff
@@ -210,7 +210,7 @@ Representation initFromConstraints(std::vector<CutCreator::DefiningConstaint> co
         }
     }
 
-    auto rows = constraints.size();
+    auto rows = static_cast<uint32_t>(constraints.size());
```

Comment:
```suggestion
    uint32_t rows = constraints.size();
```

Reply:
Sounds good to me.  Hopefully also to all the compilers :)